base-theme/layouts/partials/header.html

```diff
@@ -31,7 +31,7 @@
         id="ocw-logo-white-v2" 
         class="d-none"
         width="250"
```

Comment:
Any reason to use the width attribute here in addition to the css style above?

Reply:
This is here because the mobile logo is positioned with absolute positioning. The desktop logo doesn't need it.